pthread/pthread.c

```diff
@@ -22,7 +22,7 @@
 #include <pthread.h>
 
 
-#define CEIL(value, size)	((((value) + (size) - 1) / (size)) * (size))
+#define CEIL(value, size) ((((value) + (size)-1) / (size)) * (size))
```

Comment:
clang-format is contesting our style in macros, for some reason, I guess
```suggestion
#define CEIL(value, size) ((((value) + (size) - 1) / (size)) * (size))
```